djhtml/modes.py

```diff
@@ -289,16 +294,18 @@ class DjCSS(DjTXT):
         r"</style>",
         r"{",
         r"}",
+        r"\(",
+        r"\)",
```

Comment:
Could you replace the previous 4 lines by just:

```
r"[\{\(\)\}]",
```

That would make it more similar to DjJS mode and accomplish the same thing.